doc-content/enterprise-only/employee-rostering/optashift-ER-modifying-sample-weekend-proc.adoc

```diff
@@ -8,48 +8,32 @@ You can review the full change and apply it by using the ??? patch file, availab
 
 .Procedure
 
-. Add the necessary calculations for time slots as static methods in the `src/main/resources/org/optaplanner/openshift/employeerostering/shared/timeslot/TimeSlot.java` class of the `optashift-employee-rostering-shared` package.
-.. Add a method to determine whether the time slot happens during a weekend:
+. Add the determination of whether the shift is a weekend shift to the `src/main/java/org/optaweb/employeerostering/shared/shift/Shift.java` class of the `employee-rostering-shared` package. Add the following method:
```

Comment:
This is either two steps, or combine, something like:
...package by adding the following method:

Reply:
Disagree, but the file is not used in the document anyway because the example failed to work and Christopher could not make it work in time. Handle in BXMSDOC-3285